src/fluent_api/mod.rs

```diff
@@ -41,33 +41,41 @@ where
         Self { db }
     }
 
+    /// Select a document from the database
     #[inline]
     pub fn select(self) -> FirestoreSelectInitialBuilder<'a, D> {
         FirestoreSelectInitialBuilder::new(self.db)
     }
 
+    /// Insert a new document into the database
     #[inline]
     pub fn insert(self) -> FirestoreInsertInitialBuilder<'a, D> {
         FirestoreInsertInitialBuilder::new(self.db)
     }
 
+    /// Update a document in the database
     #[inline]
     pub fn update(self) -> FirestoreUpdateInitialBuilder<'a, D> {
         FirestoreUpdateInitialBuilder::new(self.db)
     }
 
+    /// Delete a document in the database
     #[inline]
     pub fn delete(self) -> FirestoreDeleteInitialBuilder<'a, D> {
         FirestoreDeleteInitialBuilder::new(self.db)
     }
 
+    /// List documents in the database
```

Comment:
Probably worth to mention here, that `listing` is can be used to receive all documents in the collection (or nested collection), but to specify additional query/filters you need to use `select` which is more powerful tool.